src/jdk.crypto.mscapi/windows/native/libsunmscapi/security.cpp

```diff
@@ -541,7 +541,7 @@ JNIEXPORT void JNICALL Java_sun_security_mscapi_CKeyStore_loadKeysOrCertificateC
                 else
                 {
                     if ((dwKeySpec & CERT_NCRYPT_KEY_SPEC) == CERT_NCRYPT_KEY_SPEC) {
-                        PP("CNG %I64d", (__int64)hCryptProv);
+                        PP("CNG %lld", (long long) hCryptProv);
```

Comment:
What error were you seeing here before your changes?

Reply:
Oh, that one wasn't really an error, but given that Microsoft has changed their definitions of a 64 bit int to a long long and the I64 specifier to ll in the ucrt for quite some time now, I figured that this was some extra cleanup I could fit in here. Not strictly required, but convenient code cleanliness for good measure nonetheless